README.md

```diff
@@ -8,6 +8,17 @@ Stack: TypeScript, ESLINT, Babel, Express
 - lint automatic fix: **npm run lint:fix**
 - test: **npm run test**
 
+## Initial Setup
+- Create a .env file at the root directory of the repo
+- Add **NODE_ENV=development** to the new .env file
+- Add **JWT_PRIVATE_KEY=ezfqe.QEMLKjghtrimblak.ejrkqfm__dqd2345632** to the new .env file
```

Comment:
Remove this, private keys cannot be share on a repository! ;)

Reply:
Oops, thought that was just random button mashing.